# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/cron/no_local_prototypes.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./externs.h.orig	Mon Feb 28 21:39:19 1994
+++ ./externs.h	Fri Dec 31 10:08:03 1999
@@ -109,11 +109,11 @@
 #endif
 
 #ifdef NEED_STRCASECMP
-extern	int		strcasecmp __P((char *, char *));
+// extern	int		strcasecmp __P((char *, char *));
 #endif
 
 #ifdef NEED_STRDUP
-extern	char		*strdup __P((char *));
+// extern	char		*strdup __P((char *));
 #endif
 
 #ifdef NEED_STRERROR
--- ./compat.c.orig	Mon Feb 28 21:39:14 1994
+++ ./compat.c	Fri Dec 31 10:08:50 1999
@@ -65,7 +65,7 @@
 strerror(error)
 	int error;
 {
-	extern char *sys_errlist[];
+	// extern char *sys_errlist[];
 	extern int sys_nerr;
 	static char buf[32];
 
